src/meta/src/hummock/manager/mod.rs

```diff
@@ -822,6 +822,13 @@ where
             Some(task) => task,
         };
         compact_task.watermark = watermark;
+        // to get all relational table_id from sst_info
```

Comment:
how about add some comments to explain 

`|group| group.member_table_ids != compact_task.existing_table_ids`